x-pack/legacy/plugins/maps/public/layers/sources/es_source.js

```diff
@@ -247,21 +255,25 @@ export class AbstractESSource extends AbstractVectorSource {
       return null;
     }
 
+    // fieldName could be an aggregation so it needs to be unpacked to expose raw field.
+    const rawFieldName = this._getRawFieldName(fieldName);
```

Comment:
Don't really need `_getRawFieldName ` any more. At this point, you have the metric field so why ask to find it again. Maybe just keep the original logic, its a little clearer than hiding that behind a function call

```
const realFieldName = metricField
      ? metricField.field
      : fieldName;
```

```